### PR TITLE
Update remove-former-employee-step-1.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed navigation to reflect new exchange admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5377

File: microsoft-365/admin/add-users/remove-former-employee-step-1.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ For more information about how long it takes to get someone out of email, see [W
 
 If you have email as part of your Microsoft 365 subscription, sign in to the Exchange admin center and follow these steps to block your former employee from accessing their email.
   
-1. Go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center</a>.
+1. Go to the <a href="https://admin.exchange.microsoft.com/" target="_blank">Exchange admin center</a>.
 2. In the Exchange admin center, navigate to **Recipients** \> **Mailboxes**.
-3. Double-click the user and go to the **Mailbox features** page. Under **Mobile Devices**, select **Disable Exchange ActiveSync** and **Disable OWA for Devices,** and answer **Yes** to both when prompted.
-4. Under **Email Connectivity**, select **Disable** and answer **Yes** when prompted.
+3. Double-click the user and go to the **Manage email apps settings** under **Email apps**. Turn **Off** the slider for all the options **Mobile (Exchange ActiveSync)**, **Outlook on the web**, **Outlook desktop (MAPI)**, **Exchange web services**, **POP3**, **IMAP** and Click **Save** .
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-1.md]:
```suggestion
3. Double-click the user and go to **Manage email apps settings** under **Email apps**. Turn **Off** the slider for all the options; **Mobile (Exchange ActiveSync)**, **Outlook on the web**, **Outlook desktop (MAPI)**, **Exchange web services**, **POP3**, and **IMAP**.
4. Select **Save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**